cclib/method/volume.py

```diff
@@ -253,17 +279,43 @@ def electrondensity(ccdata, volume, mocoeffslist):
                     for j, yval in enumerate(y):
                         tmp = []
                         for zval in z:
-                            tmp.append(bfs[bs].amp(xval, yval, zval))
+                            tmp.append(pyamp(bfs, bs, xval, yval, zval))
                         data[i, j, :] = tmp
                 data *= mocoeff[bs]
                 wavefn += data
             density.data += wavefn ** 2
 
-    # TODO ROHF
-    if len(mocoeffslist) == 1:
-        density.data *= 2.0
-
     return density
 
+def electrondensity(ccdata, volume, mocoeffslist):
+    """Calculate the magnitude of the electron density at every point in a volume for either up or down spin

```

Comment:
Shouldn't this be
```suggestion
    """Calculate the magnitude of the total electron density at every point in a volume
```
?

Reply:
Looks like I forgot to change the comment here. Thanks for pointing this out!